<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>561</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>OVERTIME_SYS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(21, 21, 21);
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-10</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);
border-image: url(:/icon/title.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>941</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 56, 65);
border-image: url(:/icon/backgroup.jpg);
</string>
    </property>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>441</width>
       <height>176</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>176</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: \*url();
border-top-color: rgb(179, 179, 179);
background-color: rgb(21, 103, 116);
background-color: rgb(134, 134, 134);
border-top-color: rgb(166, 166, 166);
alternate-background-color: rgb(24, 24, 24);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="output">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>150</y>
       <width>441</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>11</pointsize>
       <weight>3</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: \*url();
background-color: rgb(50, 50, 50);
color: rgb(255, 255, 255);
font: 25 11pt &quot;Ubuntu&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xlsButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>290</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:icon/save_button_han.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="wuxiuBox">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>290</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(225, 225, 225);
</string>
     </property>
     <property name="text">
      <string>午休</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fighterButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>290</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(189, 189, 189);</string>
     </property>
     <property name="text">
      <string>奋斗者</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="restButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>290</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(189, 189, 189);</string>
     </property>
     <property name="text">
      <string>周末调休</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="moneyButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(189, 189, 189);</string>
     </property>
     <property name="text">
      <string>加班换钱</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(225, 225, 225);</string>
     </property>
     <property name="text">
      <string>加班事由:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="remarkEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(50, 50, 50);border-image: \*url();color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>390</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border-image: url(:icon/start_button_han.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="e_minboBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: \*url();color: rgb(0, 0, 0);background-color: rgb(6, 252, 232);</string>
     </property>
     <item>
      <property name="text">
       <string>00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="s_minboBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: \*url();background-color: rgb(6, 252, 232);
color: rgb(0, 0, 0);
</string>
     </property>
     <item>
      <property name="text">
       <string>00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="s_hourboBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: \*url();background-color: rgb(6, 252, 232);
color: rgb(0, 0, 0);</string>
     </property>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="e_hourboBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: \*url();color: rgb(0, 0, 0);background-color: rgb(6, 252, 232);</string>
     </property>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 222, 222);</string>
     </property>
     <property name="text">
      <string>结束时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 222, 222);</string>
     </property>
     <property name="text">
      <string>开始时间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>390</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/icon/exit_button_han.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="readButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>340</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/icon/read_button_han.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="yearspinBox">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>292</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border-bottom-color: rgb(104, 104, 104);
background-image: url(:/icon/icon/yuan.png);
</string>
     </property>
     <property name="maximum">
      <number>2020</number>
     </property>
     <property name="value">
      <number>2016</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="monthspinBox">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>292</y>
       <width>44</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border-bottom-color: rgb(104, 104, 104);
background-image: url(:/icon/icon/yuan.png);
</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="detialBox">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>350</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(225, 225, 225);

</string>
     </property>
     <property name="text">
      <string>图形化显示</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>390</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/icon/delete.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>441</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: \*url();
color: rgb(0, 0, 0);
background-color: rgb(163, 163, 163);</string>
     </property>
     <property name="rowCount">
      <number>32</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>81</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>21</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>23</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>开始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>离开时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>加班时</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
    </widget>
    <zorder>yearspinBox</zorder>
    <zorder>monthspinBox</zorder>
    <zorder>calendarWidget</zorder>
    <zorder>output</zorder>
    <zorder>xlsButton</zorder>
    <zorder>wuxiuBox</zorder>
    <zorder>fighterButton</zorder>
    <zorder>restButton</zorder>
    <zorder>moneyButton</zorder>
    <zorder>label_2</zorder>
    <zorder>remarkEdit</zorder>
    <zorder>startButton</zorder>
    <zorder>e_minboBox</zorder>
    <zorder>s_minboBox</zorder>
    <zorder>s_hourboBox</zorder>
    <zorder>e_hourboBox</zorder>
    <zorder>label_6</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButton</zorder>
    <zorder>readButton</zorder>
    <zorder>detialBox</zorder>
    <zorder>deleteButton</zorder>
    <zorder>table</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="backgroup.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
